decidim-core/app/helpers/decidim/tooltip_helper.rb

```diff
@@ -4,9 +4,10 @@ module Decidim
   # This helper includes some methods to help with the inclusion of tooltips
   # on the layout.
   module TooltipHelper
-    def with_tooltip(title, &)
-      content_tag(:span, data: { tooltip: true, disable_hover: false, keep_on_hover: true, click_open: false },
-                         title:, &)
+    def with_tooltip(opts = {}, &)
```

Comment:
Could you please preserve the method signature for this method, i.e. something like:

```ruby
  def with_tooltip(title, opts = {}, &)
    # ...
  end
```

Now this change is breaking all the other uses of this method which is also causing errors in the tests / other views right now (see the CI logs).

With such changes, it would be good to keep the APIs similar, especially if there is no reason to change the signature (other than adding the extra options which can be done transparently here).

Reply:
The current uses of this helper, as I said in my previous comment, it comes from the M-type cards (deprecated) and proposals (not yet redesigned). We'll face it later on.

Reply:
OK, so you want to merge this with the failing specs? Everywhere where this is used right now in the code, it uses the old syntax.

My comment was not only referring to the current uses of this helper but also to any existing 3rd party code that may be affected by this change. This kind of API changes make the upgrades generally more difficult.

Reply:
Resolved by a8143a84898a69bd83976d63a2e704d7c5b4d28b.